### DISCUSSION TITLE
Mantine Dates: Calendar - add week view.

### DISCUSSION THREAD

**Discussion by muhammadyusuf-kurbonov** [mantinedev/mantine#3749] [Feature requests]
I need the simple week view. And, I think, the Calendar component matches the best. 

![image](https://user-images.githubusercontent.com/45625440/224467426-7b9b8a29-74e1-48ff-8059-2fe3ede39f0f.png)

Something like this

**Comment by rtivital** [comment]
What is week view? It is not clear from the screenshot

**Comment by muhammadyusuf-kurbonov** [comment]
Only a single line.
![image](https://user-images.githubusercontent.com/45625440/224489936-de6814ef-6728-4732-b784-6c6e28ae92a1.png)
Here is complete example context for component. Sorry for incomplete description

**Reply by rtivital** [reply]
I do not really understand what that means. Why are there 7 controls?
